tests/src/kinAccount.unit.test.ts

```diff
@@ -430,9 +632,10 @@ function mockCreateAccountResponse() {
 }
 
 function mockSendKinResponse(url?: string) {
-	nock(url ? url : fakeUrl)
+	const localUrl = url ? url : horizonUrl;
+	nock(localUrl)
 		.matchHeader(headerKey, headerVal)
-		.post(url => url.includes("/transactions"), "tx=AAAAAG809%2BMhGZ82rRmsoUDGVlkQGcjGXbF2fX62aTPsrih8AAAAZAAWczYAAAAHAAAAAAAAAAEAAA" +
+		.post(localUrl => localUrl.includes("/transactions"), "tx=AAAAAG809%2BMhGZ82rRmsoUDGVlkQGcjGXbF2fX62aTPsrih8AAAAZAAWczYAAAAHAAAAAAAAAAEAAA" +
```

Comment:
if there was a problem that local uri var inside the function was hidden bt the method param, then this will not solve it as we end up with the same name.